qmp/socket.go

```diff
@@ -68,6 +68,31 @@ func NewSocketMonitor(network, addr string, timeout time.Duration) (*SocketMonit
 	return mon, nil
 }
 
+// Listen creates a new SocketMonitor by listening to the provided socket file or address.
```

Comment:
I'd clarify with: "... listening for a single connection to the provided ...".

Reply:
sounds good to me